src/main.cpp

```diff
@@ -52,6 +56,15 @@ int main(int argc, char* argv[])
     QGuiApplication::setOrganizationDomain("librumreader.com");
     QGuiApplication::setApplicationName("Librum");
     QQuickStyle::setStyle(QStringLiteral("Default"));
+		
+	// store host Url to settings
+	QSettings settings;
+	QString cfgFile = settings.value("serverHost",QVariant(QString()) ).toString();
+	if (cfgFile.isEmpty()){	 	
+		settings.setValue("serverHost","https://api.librumreader.com");
+	} 	 	
+	QTextStream(stdout)<< "Host value is " << settings.value("serverHost").toString() << Qt::endl;
```

Comment:
What is the reason of using this instead of `qDebug() << "Host value is... \n";` ?